docs/references/javascript/clerk/clerk.mdx

```diff
@@ -230,4 +232,4 @@ In addition to the methods listed above, the `Clerk` class also has the followin
 [user-ref]: /docs/references/javascript/user/user
 [organization-ref]: /docs/references/javascript/organization/organization

```

Comment:
These links indicate we're using multiple reference links. As per [Styleguide 1.3, let's dedupe the reference links.](https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#13-de-dupe-reference-links)

Reply:
Also, I agree with a "Security and Privacy" section, and this page can go under that

Reply:
It's not an urgent matter, but something we should apply moving forward when accepting PRs.

Reply:
@BRKalow mind moving this to "Security" in the sidebar and renaming that item "Security and Privacy?"